dips/refunds-addendum.md

```diff
@@ -0,0 +1,30 @@
+# Diem Refunds
+
+Diem has two approaches to refunds, Refund transactions and refunds via a payment in reverse. Refund transactions primarily exist for payment violations, e.g., the recipient has no knowledge about the metadata contained within the transaction.
```

Comment:
this paragraph is sorta confusing. 
1. `Refund transactions` and `refunds via a payment in reverse` really sound like the same thing 
2. We say there are two but we only talk about one type.
3. `primarily exist for payment violations` this may be true but conflicts with the proposed user initialized refund reasons. I think it's safe to not talk about this in this PR

Reply:
> We say there are two but we only talk about one type.

This is a good but tricky point, in that, travel rule won't ever use a refund.

Reply:
what does that mean? Travel rule can have refund. check `original_payment_reference_id`